### ISSUE TITLE
unused variable, false positive, COLLECT

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1714]
`COLLECT metric INTO metrics_current_run.`

**Comment by larshp** [comment]
```abap
DATA tab TYPE STANDARD TABLE OF i WITH DEFAULT KEY.
DATA var TYPE i.
COLLECT var INTO tab.
```
